### PR TITLE
Add prefix variable

### PR DESCRIPTION
**Pull request by weirdwiz**
Add `PREFIX` to Makefile

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,9 +7,9 @@ ifeq ($(shell go help mod >/dev/null 2>&1 && echo true), true)
 	GO_BUILD=GO111MODULE=on $(GO) build -mod=vendor
 endif
 DESTDIR ?=
+PREFIX ?= /usr
```
Comment (rhatdan) [inline Makefile]: Should be /usr/local to match defaults, I believe.

Comment (weirdwiz) [reply Makefile]: Podman looks for the hooks in `/usr/share/containers/oci/hooks.d` or `/etc/containers/oci/hooks.d` . If the hooks are placed in `/usr/local/share/containers/oci/hooks.d` they won't run, I think.

https://github.com/containers/libpod/blob/master/pkg/hooks/hooks.go#L23

Comment (rhatdan) [reply Makefile]: Ok, Usually the default is not to effect the system files, but in this case we don't have much choice.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**